qiita_db/metadata_template/base_metadata_template.py

```diff
@@ -1044,7 +985,7 @@ def to_dataframe(self):
             meta = qdb.sql_connection.TRN.execute_fetchindex()
 
             # Create the dataframe and clean it up a bit
-            df = pd.DataFrame((list(x) for x in meta), columns=cols)
+            df = pd.DataFrame((list(x) for x in meta), columns=cols, dtype=str)
```

Comment:
Ideally, all `NaN`s should be `None`s, but pandas for whatever reason it's applying some kind of transformation although you're passing `dtype=str`. I think the only option would be to force everything to be `None` afterwards using this:
```python
In [32]: df
Out[32]: 
    c1 c2
0    1  3
1  NaN  2

In [33]: df.where(pd.notnull(df), None)
Out[33]: 
     c1 c2
0     1  3
1  None  2
```